src/global.ts

```diff
@@ -0,0 +1,16 @@
+export {}
```

Comment:
If we are only using this file to define types, we can use a `.d.ts` instead of `.ts`

Reply:
It's also defined [here](https://github.com/witnet/sheikah/pull/1880/files#diff-619cc2239a7eb4ef05041767e6ab0e4fb069e98ee9d2c83c2d8e1865c475e141R10), can we remove this?